src/lizaalert/courses/admin.py

```diff
@@ -278,3 +286,23 @@ class SubscriptionAdmin(admin.ModelAdmin):
     ]
 
     ordering = ("-updated_at",)
+
+
+@admin.register(Division)
+class DivisionAdmin(admin.ModelAdmin):
+    """Aдминка для Division."""
+
+    ordering = ("-updated_at",)
+    list_display = ("title", "author", "created_at", "updated_at", "courses")
```

Comment:
поле updated_at дублирует created

```suggestion
    list_display = ("title", "author", "updated_at", "courses")
```

Reply:
исправил